modules/ROOT/pages/changelog.adoc

```diff
@@ -4,66 +4,80 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 7.1.1 - 2024-05-20
+
+=== Fixed
+
+* Deleting into a list from a paragraph that has an `img` tag could cause extra inline styles to be added.
+// #TINY-10892
+* Resolved an issue where emojis were not loading correctly due to a broken CDN.
+// #TINY-10878
+* Iframes in dialogs were not rendering rounded borders correctly.
+// #TINY-10901
+* Autocompleter possible values are no longer capped at a length of 10.
+// #TINY-10942
+
 == 7.1 - 2024-05-08
 
-### Added
+=== Added
```

Comment:
I assume that this was a mistake that we made on the previous 7.1 release?

Reply:
yeah, just a `asciidoc` vs `md` syntax error.